ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetObject.java

```diff
@@ -69,17 +79,29 @@ public CliCommand init(final Arguments args) throws Exception {
     @Override
     public GetObjectResult call() throws Exception {
         try {
+            final Ds3ClientHelpers helpers = getClientHelpers();
             final Path filePath = Paths.get(prefix, objectName);
             LOG.info("Output path: " + filePath.toString());
 
-            getFileUtils().createDirectories(filePath.getParent());
-            final Ds3ClientHelpers helpers = getClientHelpers();
-            final List<Ds3Object> ds3ObjectList = Lists.newArrayList(new Ds3Object(objectName));
-
-            final Ds3ClientHelpers.Job job = helpers.startReadJob(bucketName, ds3ObjectList);
-
-            job.transfer(new FileObjectGetter(Paths.get(prefix)));
+            final Ds3Object ds3Obj = new Ds3Object(objectName.replace("\\", "/"));
```

Comment:
This seems like it can be split out into 1) a function to get the specific file we are interested and 2) actually perform the transfer.